internal/pkg/cli/pipeline_init.go

```diff
@@ -20,36 +25,39 @@ import (
 	"github.com/aws/copilot-cli/internal/pkg/deploy/cloudformation"
 	"github.com/aws/copilot-cli/internal/pkg/manifest"
 	"github.com/aws/copilot-cli/internal/pkg/template"
-	"github.com/aws/copilot-cli/internal/pkg/term/color"
 	"github.com/aws/copilot-cli/internal/pkg/term/command"
-	"github.com/aws/copilot-cli/internal/pkg/term/log"
 	"github.com/aws/copilot-cli/internal/pkg/term/prompt"
-	"github.com/aws/copilot-cli/internal/pkg/version"
 	"github.com/aws/copilot-cli/internal/pkg/workspace"
 	"github.com/spf13/afero"
-	"github.com/spf13/cobra"
 )
 
 const (
 	pipelineSelectEnvPrompt     = "Which environment would you like to add to your pipeline?"
 	pipelineSelectEnvHelpPrompt = "Adds an environment that corresponds to a deployment stage in your pipeline. Environments are added sequentially."
 
-	pipelineSelectGitHubURLPrompt     = "Which GitHub repository would you like to use for your service?"
-	pipelineSelectGitHubURLHelpPrompt = `The GitHub repository linked to your workspace.
+	pipelineSelectURLPrompt     = "Which repository would you like to use for your pipeline?"
+	pipelineSelectURLHelpPrompt = `The repository linked to your pipeline.
 Pushing to this repository will trigger your pipeline build stage.
 Please enter full repository URL, e.g. "https://github.com/myCompany/myRepo", or the owner/rep, e.g. "myCompany/myRepo"`
 )
 
 const (
 	buildspecTemplatePath = "cicd/buildspec.yml"
 	githubURL             = "github.com"
-	defaultBranch         = "main"
+	ccIdentifier          = "codecommit"
+	awsURL                = "aws.amazon.com"
+	ghProviderName        = "GitHub"
+	ccProviderName        = "CodeCommit"
+	defaultGHBranch       = "main"
+	defaultCCBranch       = "master"
 )
 
 const (
-	fmtSecretName       = "github-token-%s-%s"
-	fmtPipelineName     = "pipeline-%s-%s-%s"
-	fmtPipelineProvider = "https://%s/%s/%s"
+	fmtSecretName         = "github-token-%s-%s"
+	fmtGHPipelineName     = "pipeline-%s-%s-%s"
+	fmtCCPipelineName     = "pipeline-%s-%s"
+	fmtGHPipelineProvider = "https://%s/%s/%s"
+	fmtCCPipelineProvider = "https://%s.console.%s/codesuite/codecommit/repositories/%s"
```

Comment:
What do you think of?
```suggestion
	fmtGHRepoURL = "https://%s/%s/%s"
	fmtCCRepoURL = "https://%s.console.%s/codesuite/codecommit/repositories/%s"
```

Reply:
Ah, much better. Done.